source/help/apps/desktop-changelog.rst

```diff
@@ -1,8 +1,119 @@
 Desktop Application Changelog
-=============================
+========================================
+
+Release v3.5.0
+--------------
+
+Release date: December 14, 2016
+
+Improvements
+~~~~~~~~~~~~
+
+All Platforms
+^^^^^^^^^^^^^
+
+-  URL address is shown when hovering over links with a mouse
+-  Added CTRL+SHIFT+MINUS as a shortcut for decreasing font size
+   (zooming out)
+-  Reduce upgrade issues by properly clearing cache when updating the
+   desktop app to a new version (the application cache will be purged
+   whenever the desktop app version changes)
+-  When disconnected from Mattermost, the "Cannot connect to Mattermost"
+   page is now properly aligned at the top of the window
+-  Suppressed error messages when launching the app from the command
+   line and ``certificate.json`` is missing in the user data directory
+
+Windows
+^^^^^^^
+
+-  Link addresses can now be copied and pasted inside the app
+
+Bug Fixes
+~~~~~~~~~
+
+All Platforms
+^^^^^^^^^^^^^
+
+-  YouTube previews now work, even if mixed content is allowed
+-  Fixed an incorrect cursor mode for "Edit" and "Remove" buttons on the
+   Settings page
+-  Fixed an issue where "Zoom in/out" settings did not properly work
+
+Windows
+^^^^^^^
+
+-  The menu bar option for "Redo" is now properly shown as CTRL+Y
+
+Mac
+^^^
+
+-  Fixed an issue where the default download folder was ``Macintosh HD``
+-  Removed an unexpected "Show Tab Bar" menu item on macOS 10.12
+
+Linux (Beta)
+^^^^^^^^^^^^
+
+-  Fixed an issue where the option "Leave app running in notification
+   area when the window is closed" was never enabled.
+
+Known Issues
+~~~~~~~~~~~~
+
+All Platforms
+^^^^^^^^^^^^^
+
+-  `If you click twice on the tab bar, and then attempt to use the "Zoom
+   in/out" to change font size, the app window doesn't render
+   properly <https://github.com/mattermost/desktop/issues/334>`__
+-  `Direct messages cause notification icons to appear on each team on
+   the tab bar, which don't clear until you click on each
+   team <https://github.com/mattermost/desktop/issues/160>`__
+-  `After right-clicking an image or a link, the "Copy Link" option
+   sometimes moves around when clicking different places
+   afterwards <https://github.com/mattermost/desktop/issues/340>`__
+
+Windows
+^^^^^^^
+
+-  [Windows 7] `Sometimes the app tries to render the page inside the
```

Comment:
Propose: "...tries to render the page inside the app instead of in a new browser tab when clicking links" > "tries to render clicked linked inside the app, instead of in a new browser tab"